airflow/providers/yandex/hooks/yandex.py

```diff
@@ -167,10 +168,8 @@ def _get_credentials(self) -> dict[str, Any]:
         if service_account_key:
             return {"service_account_key": service_account_key}
 
-        raise AirflowException(
-            "No credentials are found in connection. Specify either service account "
-            "authentication JSON or user OAuth token in Yandex.Cloud connection"
-        )
+        logging.info("using metadata service as credentials")
```

Comment:
[`BaseHook`](https://github.com/apache/airflow/blob/2bcd450e84426fd678b3fa2e4a15757af234e98a/airflow/hooks/base.py#L35) based on [`LoggingMixin`](https://github.com/apache/airflow/blob/1e1adc569f43494aabf3712b651956636c04df7f/airflow/utils/log/logging_mixin.py#L67), so you could call `self.log.<logger-method>` instead of create separate logger / use root logger

```suggestion
        self.log.info("using metadata service as credentials")
```